scripts/run-screenshot-tests.sh

```
@@ -0,0 +1,441 @@
+            fi
+            
+            # Clean up emulator
+            cleanup_emulator
+            
+            # Wait before next test
+            sleep 5
+        done
+        
+    else
+        # Run specific API level only
+        if [[ " ${API_LEVELS[*]} " =~ " ${SELECTED_API} " ]]; then
```

Shellcheck (0.10.0)

[warning] Remove quotes from right-hand side of =~ to match as a regex rather than literally.

(SC2076)